sources/settings-migrations/v1.39.0/kubelet-container-log-extra/src/main.rs

```diff
@@ -0,0 +1,21 @@
+use migration_helpers::common_migrations::AddSettingsMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added new settings for configuring the NVIDIA k8s device plugin.
+fn run() -> Result<()> {
+    migrate(AddSettingsMigration(&[
+        "settings.kubernetes.container_log_max_workers",
+        "settings.kubernetes.container_log_monitor_internal"
```

Comment:
These need to be kebab-case for API consistency.

If the values are actually accepted and saved in snake case, that's a problem that needs to be fixed in the settings struct.

Reply:
Nice catch! The setting shape is correct. And migration is what I am testing next. And I am pretty sure it will fail without the case change you just mentioned!